include/mbedtls/ssl.h

```diff
@@ -2007,13 +2020,21 @@ void mbedtls_ssl_conf_authmode( mbedtls_ssl_config *conf, int authmode );
 *                        lack of replay protection of the early data application
 *                        payloads.
 *
-* \param buffer  Early data to be transmitted
-*
-* \param len     Length of early data
+* \param max_early_data  Max number of bytes allowed for early data (server only).
+* \param early_data_callback Callback function when early data is received (server
+*                            only).
 */
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) && defined(MBEDTLS_ZERO_RTT)
-void mbedtls_ssl_set_early_data(mbedtls_ssl_context* ssl, int early_data, char* buffer, unsigned int len, int(*early_data_callback)(mbedtls_ssl_context*,
-    unsigned char*, size_t));
+void mbedtls_ssl_conf_early_data( mbedtls_ssl_config* conf, int early_data,
+                                  size_t max_early_data,
+                                  int(*early_data_callback)( mbedtls_ssl_context*,
```

Comment:
This should be const unsigned char* because the user isn't supposed to manipulate the 0-RTT buffer.